cmd/copyDownloadBlobEnumerator.go

```diff
@@ -154,7 +154,10 @@ func (e *copyDownloadBlobEnumerator) enumerate(cca *cookedCopyCmdArgs) error {
 				}
 				// check for the special character in blob relative path and get path without special character.
 				blobRelativePath = util.blobPathWOSpecialCharacters(blobRelativePath)
-
+				// force path to be AZCOPY_PATH_SEPARATOR_STRING, only really important on Windows.
+				if common.AZCOPY_PATH_SEPARATOR_STRING == `\` {
+					blobRelativePath = strings.Replace(blobRelativePath, "/", common.AZCOPY_PATH_SEPARATOR_STRING, -1)
+				}
```

Comment:
As noted elsewhere, I feel uneasy about changing AZCOPY_PATH_SEPARATOR_*.  Can you discuss with Ze please?

Reply:
As noted previously, Ze originally suggested this. It reduced a significant amount of code duplication, and reduced the amount of "hacks" I had to do in order to make it work.